package-lock.json

```diff
@@ -0,0 +1,2228 @@
+{
```

Comment:
Hmm, did you install the dependencies with `npm` instead of `yarn` at some point? The `yarn.lock` file should be committed but we should avoid including both to prevent possible inconsistencies in our dependency tree.

Reply:
Oh good catch, I definitely did.